File: backend/backup/feature-flag-service/go/internal/model/value_type.go
package model

// ValueType represents the type of a feature flag value
type ValueType string

const (
	// ValueTypeBoolean represents a boolean value
	ValueTypeBoolean ValueType = "BOOLEAN"
	
	// ValueTypeString represents a string value
	ValueTypeString ValueType = "STRING"
	
	// ValueTypeInteger represents an integer value
	ValueTypeInteger ValueType = "INTEGER"
	
	// ValueTypeDouble represents a double value
	ValueTypeDouble ValueType = "DOUBLE"
	
	// ValueTypeJSON represents a JSON value
	ValueTypeJSON ValueType = "JSON"
)

File: workspace/backend/backup/feature-flag-service/python/feature_flag_service/model/value_type.py
"""
Value type module for the Feature Flag Service.
"""

from enum import Enum, auto


class ValueType(str, Enum):
    """
    Enum for feature flag value types.
    """
    BOOLEAN = "BOOLEAN"
    STRING = "STRING"
    INTEGER = "INTEGER"
    DOUBLE = "DOUBLE"
    JSON = "JSON"

